### PR TITLE
add GetLatency func

### PR DESCRIPTION
**Pull request by edisonxiang**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
The `GetLatency` is deleted in v1.0, we need to keep it for calculating the latency metrics.
* add GetLatency func
* quic support in hubtest
* bump image version

### CODE REVIEW

**Comment by sids-b** [comment]
@edisonxiang , please squash the commits. The PR looks good to me

**Comment by edisonxiang** [comment]
@sids-b Done

File: tests/performance/common/common.go [hunk 1]
```diff
@@ -317,3 +319,29 @@ func SendHttpRequest(method, reqApi string, body io.Reader) (error, *http.Respon
 	}
 	return nil, resp
 }
+
+// GetLatency calculates latency based on different percent
+func GetLatency(pods []types.FakePod) types.Latency {
```
Comment (rohitsardesai83) [inline tests/performance/common/common.go]: where is this func used ?

Comment (edisonxiang) [reply tests/performance/common/common.go]: Hey rohit, please find it here:)
https://github.com/kubeedge/kubeedge/blob/c005656e5ab2d363640fba804c8fca49e38f161e/tests/performance/hubtest/hubtest_test.go#L50

**Pull request merged**